### DISCUSSION TITLE
Support for Content Bundles?

### DISCUSSION THREAD

**Discussion by mattmccray** [estruyf/vscode-front-matter#890] [Q&A] [answered]
Is there a way to support content bundles? For example, the content for my SSG looks like this:

```
$/
  content/
    [content-type]/
      [slug]/
        index.md
        cover.webp
```

Which might look like this:

```
$/
  content/
    posts/
      welcome-aboard/
        index.md
        cover.webp
        other-media.webp
      another-post/
        index.md
        cover.webp
```

Is there a way to configure the extension to support this structure?

**Comment by davidsneighbour** [comment, marked as answer]
`pageBundle` ;)

It's all in the name: https://frontmatter.codes/docs/content-creation/additional-config#page-and-leaf-bundles

**Reply by estruyf** [reply]
Thanks, @davidsneighbour, for providing the answer. I took Hugo's name for the `pageBundle`.

**Reply by mattmccray** [reply]
Thanks!
